package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "pethome_raffle_backend",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:AdrianMendez1199/pethome_raffle_backend.git",
+  "author": "Adrian Mendez <mendezadrian149@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "lint": "eslint . --ext .ts",
+    "remove": "rm -rf dist",
+    "build": "tsc",
+    "server": "node ./dist/index.js",
+    "ts": "ts-node ./src/index.ts",
+    "dev": "NODE_ENV=development nodemon"
+  },
+"dependencies": {
+  "@prisma/client": "^2.0.0-beta.3",
+  "bcrypt": "^4.0.1",
+  "dotenv": "^8.2.0",
+  "graphql-tools": "^5.0.0",
+  "graphql-yoga": "^1.18.3",
```

Comment:
Why did you use yoga instead of Apollo?

Reply:
just a matter of taste, in the end yoga uses apollo.
yoga is nothing more than a grouping of some tools, such as apollo-server, graphql-suscription among others. I can make the change apollo if you wish.